app/_hub/kong-inc/rate-limiting-advanced/_index.md

```diff
@@ -161,14 +170,17 @@ params:
       value_in_examples: false
       datatype: boolean
       description: |
-        Optionally hide informative response headers. Available options: `true` or `false`.
+        Optionally hide informative response headers that would otherwise
+        provide information about the current status of limits and counters as
+        described in the paragraph [Headers sent to the client](#headers-sent-to-the-client).
+        Available options: `true` or `false`.
     - name: redis.host
       required: semi
       default: null
       value_in_examples: null
       datatype: string
       description: |
-        Host to use for Redis connection when the `redis` strategy is defined.
+        Host (or IP address) to use for Redis connection when the `redis` strategy is defined.
```

Comment:
Is this saying you can have a host or IP address as the string or that the host must be an IP address?

Reply:
The former. I rephrased it to hopefully make it more clear. Please feel free to improve the wording, thank you!